README.win32.adoc

```diff
@@ -29,7 +29,8 @@ Here is a summary of the main differences between these ports:
 (*):: Cygwin-generated `.exe` files are linked with the Cygwin DLL. Since
 version 2.5.2 released in June 2016, this DLL is licensed under the LGPLv3 with
 a static linking exception; prior versions of the DLL are distributed under the
-GPL. Thus, these `.exe` files can only be distributed under compatible license.
+GPL (the cygwin version can be obtained with the `uname -r` command). Thus,
+these `.exe` files can only be distributed under a compatible license.
```

Comment:
@avsm So if I understand correctly, you must compile against 2.5.2 or later to distribute LGPL (or any licence via the linking exception) and then obviously you, and anyone else, can't distribute LGPL with a 2.5.1 DLL or earlier because that would violate the distribution clause of the DLL? I'm guessing that if you compile with 2.5.1 or earlier you must be GPL no matter what because of the import libraries (which are GPL)?

In which case, how about a wholesale re-word:
```
(*):: Executables generated by the native GCC package in Cygwin are linked with
the Cygwin DLL and require this to be distributed with your programs.
Executables generated by Microsoft Visual C++ or the Mingw-w64 compilers (even
when run in Cygwin as `i686-w64-mingw32-gcc` or `x86_64-w64-mingw32-gcc`) are
not linked against this DLL. Prior to Cygwin 2.5.2 (the Cygwin version can be
obtained with `uname -r`) the Cygwin DLL is distributed under the GPL, requiring
any programs linked with it to be distributed under a compatible licence. Since
version 2.5.2, the Cygwin DLL is distributed under the LGPLv3 with a static
linking exception meaning that, like executables generated by Microsoft Visual
C++ or the Mingw-w64 compilers, generated executables may be distributed under
terms of your choosing.
```